psignifit/likelihood.py

```diff
@@ -41,7 +41,12 @@ def posterior_grid(data, sigmoid: Sigmoid, priors: Dict[str, Prior],
             grid_max[name] = None
         else:
             grid_max[name] = grid_values[index]
-    return p, grid_max
+
+    # normalize the posterior_grid
+    posterior_volumes = p * integral_weights([grid_value for _, grid_value in sorted(grid.items())])
+    posterior_integral = posterior_volumes.sum()
```

Comment:
Is this sum safe? Are we summing too many little numbers with some big numbers and so losing precision?